index.html

```diff
@@ -44,7 +44,7 @@ <h1 class="header_h1">Erica L. Winberry</h1>
     </header>
     <div class="clearfix"></div>
     <main>
-      <div id="text-container">
+      <div id="text-container"> <!-- Use this to fix background? -->
```

Comment:
I'm still having problems with how the text displays over the dark part of the background at the bottom. Need to somehow get that text more visible--either by making the background lighter, or figuring out how to change the text color. I haven't found a way to make a text gradient color with an image in a background. Maybe ask Brian for advice, if he's around?